querydsl-collections/src/test/java/com/querydsl/collections/NumberTest.java

```diff
@@ -15,8 +15,8 @@
     @Test
     public void sum() throws Exception {
         NumberPath<BigDecimal> num = Expressions.numberPath(BigDecimal.class, "num");
-        CollQuery<?> query = CollQueryFactory.from(num, Arrays.asList(new BigDecimal("1.6"), new BigDecimal("1.3")));
+        CollQuery<BigDecimal> query = CollQueryFactory.<BigDecimal> from(num, Arrays.<BigDecimal> asList(new BigDecimal("1.6"), new BigDecimal("1.3")));
```

Comment:
`CollQueryFactory` suffers from https://stackoverflow.com/a/32294457/2104280.

This is technically not a breaking API change: users that already were on Java 8, struggled with the exact same issue in their own codebase alrady.